java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1255,6 +1256,30 @@ public void sadd_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void sismember_returns_success() {
+        // setup
+        String key = "testKey";
+        String member = "testMember";
+        String[] arguments = new String[] {key, member};
+
+        CompletableFuture<Boolean> testResponse = new CompletableFuture<>();
+        testResponse.complete(true);
+
+        // match on protobuf request
+        when(commandManager.<Boolean>submitNewCommand(eq(SIsMember), eq(arguments), any()))
+                .thenReturn(testResponse);
+
+        // exercise
+        CompletableFuture<Boolean> response = service.sismember(key, member);
+        Boolean payload = response.get();
```

Comment:
nit
```suggestion
        boolean payload = response.get();
```

Reply:
Why? The return type is Boolean should it be primitive?

Reply:
Could be both - up to you

Reply:
Why unBox? 
There's no need to convert the value from boolean to Boolean if we update the assert below to match a Boolean value (instead of the primitive `true`).